central/scannerv4definitions/handler/mappingUpdater.go

```diff
@@ -0,0 +1,123 @@
+package handler
```

Comment:
this is all very similar to `central/scannerdefinitions/handler/updater.go`. If we do end up just overloading that endpoint and moving everything over to `central/scannerdefinitions`, I think we can just reuse the pre-existing updater code and just generalize it